### DISCUSSION TITLE
Blueprint navigation

### DISCUSSION THREAD

**Discussion by Megurine** [microdee/OpenUnrealSpaceMouse#26] [Ideas]
Is there any possibilities about moving in the blueprint editor with the SpaceMouse ? (X/Y/Zoom)

It would be awesome for developpers like me.

Thank you !

**Comment by litruv** [comment]
https://github.com/user-attachments/assets/27eb6136-523e-446c-b1bc-7f1d02018d0b

It's been just over 5 years, we have the technology!! ~~Cleaning up a branch for use soon 👍~~ Check PR's

**Reply by microdee** [reply]
so awesome! you're a legend, thanks for the PR as well!

**Comment by microdee** [comment]
@litruv [I'm making the 1.3 release](https://github.com/microdee/OpenUnrealSpaceMouse/pull/113) with this feature included. Similar to the ortho-viewport navigation I've added the user preference for selecting the axis of zoom (push forward/backward to zoom vs. push downward/upward to zoom) ([see this commit](https://github.com/microdee/OpenUnrealSpaceMouse/pull/113/changes/7a759b29bf92ab4108d937b35a055fda24589526)). I will start porting this to the new 2.0 architecture shortly.
